pages/index.js

```diff
@@ -12,11 +29,20 @@ export default function HomePage({ outivities, favorites, onToggleFavorite }) {
       <StyledTitle>All Outivities</StyledTitle>
```

Comment:
I don't think we need to style things inside the Head component from next, as they will be used in the <head> of the document. 

[See here](https://nextjs.org/docs/pages/api-reference/components/head).

Reply:
We think that you maybe mixed something up here? 🤓

There is a `<title>` (unstyled) inside the `<Head>` and a `<StyledTitle>` which is actually the header of the current page.